apps/server/src/modules/authentication/mapper/current-user.mapper.ts

```diff
@@ -3,44 +3,21 @@ import { Role, User } from '@shared/domain';
 import { UserDO } from '@shared/domain/domainobject/user.do';
 import { RoleDto } from '../../role/service/dto/role.dto';
 import { ICurrentUser } from '../interface';
+import { JwtPayload } from '../interface/jwt-payload';
 
 export class CurrentUserMapper {
 	static userToICurrentUser(accountId: string, user: User, systemId?: string): ICurrentUser {
-		const permissions = new Set<string>();
-		if (user.roles) {
-			user.roles.getItems().forEach((role) => {
-				if (role.permissions) {
-					role.permissions.forEach((permission) => {
-						permissions.add(permission.toString());
-					});
-				}
-			});
-		}
 		return {
 			accountId,
 			systemId,
-			roles: user.roles.getItems().map((role: Role) => role.name),
+			roles: user.roles.getItems().map((role: Role) => role.id),
```

Comment:
This change has no impacts?

Reply:
Feathers used role IDs in the JWT. Nest used role names. I changed it in nest to keep it the way we had it for way longer time.
Since the change in Nest did not cause any errors, it seem like these roles are not used at all. Capcakes asked to have IDs

Reply:
ok